prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -92,6 +92,11 @@ spec:
               cpu: 10m
           securityContext:
             privileged: true
+          env:
```

Comment:
No longer applies, as the PR is no longer changing the YAML files and relies on `HOSTNAME` instead.